src/ansys/additive/core/machine.py

```diff
@@ -316,6 +410,38 @@ def slicing_stripe_width(self, value: float):
         )
         self._slicing_stripe_width = value
 
+    @property
+    def heat_source_model(self) -> str:
+        """Type of heat source model used in simulations.
+        The heat source model defines how the laser beam is modeled in the simulation.
+
+        Valid values are "gaussian" and "ring".
+        """
+        return self._heat_source_model
+
+    @heat_source_model.setter
+    def heat_source_model(self, value: str):
+        self.__convert_heat_source_model_str(value)
+        self._heat_source_model = value
+
+    @property
+    def ring_mode_coefficient_set_index(self) -> int:
+        """Ring mode coefficient set index.
```

Comment:
Here too, we probably need a link to any docs we have describing the parameter sets.